package.json

```diff
@@ -69,7 +69,7 @@
     "nyc": "^15.0.0",
     "pkg-up": "^4.0.0",
     "sinon": "^17.0.0",
-    "typedoc": "^0.25.4",
+    "typedoc": "~0.25.4",
```

Comment:
TypeDoc introduces breaking changes between SemVer minor versions (which is compliant since it has major version 0), so let's pin this to the same minor line.

Reply:
FWIW since it's pre-1.0, I think package managers do what you're asserting anyway.